### PR TITLE
DB: Faceless Lurker drop

### PR DESCRIPTION
**Pull request by poszer**
The creature Faceless Lurker should drop Frostweave Cloth with drop chance 100

##### CHANGES PROPOSED:

-  Added missing row in DB
-  

###### ISSUES ADDRESSED:

Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1539

##### TESTS PERFORMED:

Tested, sql is imported without any issue. Tested in game, the listed creature now drop the item.

##### HOW TO TEST THE CHANGES:

1. .go creature 203343
2. Kill him, Loot it. It should drop frostweave cloth 1-7 with drop chance 100%

##### KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

##### Target branch(es):

Master

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1552859629415687000.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1552859629415687000');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (31691, 33470, 0, 100, 0, 1, 0, 1, 7, NULL);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1552859629415687000.sql]: Always `DELETE` before insert in order to make your query re-executable :)

In your case, something like

```sql
DELETE FROM `creature_loot_template` WHERE `Entry` = 31691 AND `Item` = 33470;
```

**Comment by chendh123** [comment]
Why is it not effective to use two references?
What's wrong with the original configuration?
creature_loot_template
image
reference_loot_template
image

**Comment by Stoabrogga** [comment]
There seems to be a condition which only allows the drop of "Frostweave Cloth" (33470) with loot reference ID 26041 if the player has the aura "Northern Cloth Scavenging" (59390). Just look into the conditions table for SourceEntry 33470. I tested this successfully with applied aura 59390, so I don't think this needs to be fixed at all.

**Comment by poszer** [comment]
Than we will close this, thanks @Stoabrogga

**Pull request closed without merging**